webpack/assets/javascripts/react_app/components/componentRegistry.js

```diff
@@ -141,7 +142,7 @@ const coreComponets = [
   { name: 'SettingUpdateModal', type: SettingUpdateModal },
   { name: 'PersonalAccessTokens', type: PersonalAccessTokens },
   { name: 'ClipboardCopy', type: ClipboardCopy },
-
+  { name: 'MemoryAllocationInput', type: MemoryAllocationInput },
```

Comment:
I believe this is exactly the type of component that should have `store: false`.